app/assets/stylesheets/psique/foundation/_tools.scss

```diff
@@ -10,6 +10,7 @@
 // -------------------------------------
 
 .poar {
```

Comment:
This class mean 'position absolute right', with these changes, it completely misses the point. This is wrong.